clients/java/src/test/java/io/camunda/client/OAuthCredentialsProviderTest.java

```diff
@@ -197,6 +198,28 @@ void shouldRequestTokenWithScopeAndAddToCall() throws IOException {
         .containsExactly(new Credential("Authorization", TOKEN_TYPE + " " + ACCESS_TOKEN));
   }
 
+  @Test
+  void shouldRequestTokenWithEmptyScopeAndAddToCall() throws IOException {
+    // given
+    final OAuthCredentialsProvider provider =
+        new OAuthCredentialsProviderBuilder()
+            .clientId(CLIENT_ID)
+            .clientSecret(SECRET)
+            .audience(AUDIENCE)
+            .scope("")
+            .authorizationServerUrl(tokenUrlString())
+            .credentialsCachePath(cacheFilePath.toString())
+            .build();
+    mockCredentials(ACCESS_TOKEN, "");
+
+    // when
+    provider.applyCredentials(applier);
+
+    // then
+    assertThat(applier.getCredentials())
+        .containsExactly(new Credential("Authorization", TOKEN_TYPE + " " + ACCESS_TOKEN));
```

Comment:
❓ we are not explicitly asserting that the empty scope is actually part of the payload, right? It seems, we rely implicitly on the mock to return a 404 if the actual payload does not match the mocked one. I guess it's fine, but makes it a bit harder to reason about the test at first glance.